vtm/src/org/oscim/layers/MapEventLayer.java

```diff
@@ -427,7 +427,7 @@ private void onActionMove(MotionEvent e) {
                 mPrevPinchWidth = pinchWidth;
 
                 if (Parameters.ANIMATOR2) {
-                    if (mDoScale && scaleBy != 1f) {
+                    if (scaleBy != 1f) {
```

Comment:
The indicator `mDoScale` not always is `true` if scales are proceeded, e.g. while scaling and rotating simultaneously. This is misleading.